Cut down word count in adv/disadv

zachartmann opened the pull request:
* Cut down to 409 from just below 600w
* Think I kept most of the information there

zachartmann committed:
Cut down word count in adv/disadv

Cut down to 409 from just below 600w
Think I kept most of the information there

zachartmann committed:
Add ideas and reflection

Trying to finish it all now
* Added ideas and reflection
* Removed unused headings

zachartmann committed:
Removed other unused lines

* Removed some lines that weren't needed
* Will probably cause some conflicts I'm happy to resolve

zacaz90 commented:
Don't miss that I did more than just cut down words, want to make sure I didn't cut anything important from my section nor from the project as a whole

seant1 commented on Report.md:
```diff
@@ -72,10 +66,17 @@ Stakeholders refer to individuals/entities that have an interest in the developm
+The following features were concluded from the brainstorming session. 
+Feature | Justification
+--- | ---
+Enhance Interactive Voice Response unit | Gain initial understanding of customer needs without the need for an in-person RM answering the call. This, in addition to the addition of hold music, were the best ideas ideated to increase customer retention in calls while no RMs were available. We concluded that other ideas such as playing ads while customers were on hold were abandoned as they would l ...
```
are we keeping "This, in addition..." derp sentence?

Aiswarya668 replied:
Can we make it shorter - In addition, hold music respective of customer demographic was suggested to increase customer retention in calls while no RMs were available. We concluded that other ideas such as playing ads while customers were on hold were abandoned as they would lead to negative customer experience.

Aiswarya668 replied:
Or we could get rid of it if it takes up space hahah can't believe we didn't catch it

zachartmann replied:
Copied it straight across ahaha good catch, probably just going to remove it - doesn't add a lot

seant1 commented on Report.md:
```diff
@@ -72,10 +66,17 @@ Stakeholders refer to individuals/entities that have an interest in the developm
+Enhance Interactive Voice Response unit | Gain initial understanding of customer needs without the need for an in-person RM answering the call. This, in addition to the addition of hold music, were the best ideas ideated to increase customer retention in calls while no RMs were available. We concluded that other ideas such as playing ads while customers were on hold were abandoned as they would l ...
+Improve RM questionnaire  | Allows better initial classification of the RM by providing more parameters for CMC to score by.
+Added internal wiki of further package information | This was decided to be the best way to have all RMs confident with all packages, so they are more likely to make a sale to customers even if they lack experience with the destination.
+Feedback took implemented  |  Customer feedback tool was added so that the CMC can implement RM score in real-time after the call is undertaken. 
```
Typos
```suggestion
Feedback tool |  Customer feedback tool was added so that the CMC could implement RM score in real-time after the call is undertaken. 
```

zachartmann committed:
Brought across the executive summary

A little surprised there were no conflicts so I brought across the summary so I don't accidentally overwrite it.
It's ripped straight from master.

zachartmann committed:
Merge branch 'master' into develop

seant1 commented on Report.md:
```diff
@@ -72,10 +66,17 @@ Stakeholders refer to individuals/entities that have an interest in the developm
+Feedback took implemented  |  Customer feedback tool was added so that the CMC can implement RM score in real-time after the call is undertaken. 
+Enhanced RM guidelines  | Natural language processing is embedded in the CMC to enhance the autogenerated RM scripts to enhance sale chances. 
+
+The biggest issue we, as a group, faced over the course of our project was our unfamiliarity with GitHub. This took out a chunk of each group meeting, as we had to spend time figuring out a correct and consistent workflow. Another issue we found was the lack of face-to-face communication due to COVID-19. This was especially prevalent when filming the video as collating everyone’s individual video ...
```
chunk --> portion (more formaliticality?)
```suggestion
The biggest issue we, as a group, faced over the course of our project was our unfamiliarity with GitHub. This took out a portion of each group meeting, as we had to spend time figuring out a correct and consistent workflow. Another issue we found was the lack of face-to-face communication due to COVID-19. This was especially prevalent when filming the video as collating everyone’s individual video and editing became very complicated.
```

seant1 commented on Report.md:
```diff
@@ -164,27 +161,21 @@ These diagrams represents the dynamic behaviour of objects and the structure of
+Lessening the call flow control/routing issues through the development of the new information system will reduce wait times for customers and provide more effective matches with RMs (Relationship Managers). This increase in user experience will improve customer retention, as they see changes being made in the system for their benefit. In addition, implementing an easier method of feedback elicita ...
 
-From a more business-centric point of view, not only will the customers be happier with the system, but the travel company will also save costs in handling the soon-to-be larger customer base. The information system's ability to reduce per-call handling time will reduce the cost of inbound calls to the CMC. This, in combination with the increased income that comes with a greater amount of custome ...
+From a more business-centric point of view, the travel company will save costs in handling the increasing customer base. This is because of the information system's ability to reduce per-call handling time, which reduces inbound call costs.
```
Minor grammatical suggestion
```suggestion
From a more business-centric point of view, the travel company will save costs in handling the increasing customer base. This is due to the information system's ability to reduce per-call handling time, which reduces inbound call costs.
```

Aiswarya668 commented on Report.md:
```diff
@@ -164,27 +160,21 @@ These diagrams represents the dynamic behaviour of objects and the structure of
 Potential Advantages and Disadvantages
 ======================================
 ## **Competitive Advantage**
-Developing this new information system, if successful, will lead to a plethora of competitive advantages against other major travel companies looking to increase their customer loyalty or retention. Currently, the major travel company involved is in a great position in the market but has some areas that it is lacking in. For example, the CMC (Call Management Centre) has reduced operating efficien ...
```
"Currently, the major travel company involved " for this maybe reword it to help the flow "Current the major travel company is in a position to invest resources into areas they are lacking in."

zachartmann replied:
I cut that whole sentence for word count, are you suggesting we bring it back in?

Aiswarya668 replied:
Oops I didn't realise it was cut out you can get rid of it

seant1 commented on Report.md:
```diff
@@ -164,27 +161,21 @@ These diagrams represents the dynamic behaviour of objects and the structure of
+If the information system is ineffective, there are some considerable adverse effects to first take into account. Ranging from the loss of customers to increased operating costs for no benefit, the adverse effects possible should be weighed into consideration alongside the competitive advantages to most accurately see the possible result of the information system.
 
-By implementing what the travel company believes is a solution to their biggest issues, and communicating to users as such, there is an inherent risk if we fail to deliver. By producing a suboptimal solution, customers may feel rightly disappointed, as their expectations were not met. Particularly if these issues have been ongoing for some time, customers who thought the end of these issues were  ...
+By communicating to users that the travel company is implementing a solution to their biggest issues, there is an inherent risk if we fail to deliver. If the solution fails to meet expectation, customers may feel rightly disappointed. If these issues have been ongoing, those who expected an end to their problems may be especially upset and cease interaction with the travel company. There is then  ...
```
More shortening (without info loss)
```suggestion
By communicating to users that the travel company is implementing a solution to their biggest issues, there is an inherent risk if we fail to deliver. If the solution fails to meet expectation, customers may feel rightly disappointed particularly if these issues have been ongoing. There is a high risk that they may cease interaction with the travel company and move on to a competitor - a huge competitive disadvantage.
```

Aiswarya668 commented on Report.md:
```diff
@@ -164,27 +160,21 @@ These diagrams represents the dynamic behaviour of objects and the structure of
+Developing this new information system, if successful, will lead to a plethora of competitive advantages against other major travel companies looking to increase their customer loyalty or retention. Currently, the CMC (Call Management Centre) has reduced operating efficiency due to call flow control and call routing issues. This leads to some customers having long wait times and drives up the inb ...
 
-If the call flow control and routing issues are removed or even lessened through the development of the new information system, then customers will have shorter wait times as well as more effective matches with RMs (Relationship Managers). This increase in user experience will improve customer retention, as they see changes being made in the system for their benefit. In addition, implementing an  ...
+Lessening the call flow control/routing issues through the development of the new information system will reduce wait times for customers and provide more effective matches with RMs (Relationship Managers). This increase in user experience will improve customer retention, as they see changes being made in the system for their benefit. In addition, implementing an easier method of feedback elicita ...
```
Maybe change this "increase" to This "enhancement" in user experience will improve.....

seant1 reviewed (changes requested):
Neat 📚. Just some minor wording suggestions

Aiswarya668 commented on Report.md:
```diff
@@ -164,27 +160,21 @@ These diagrams represents the dynamic behaviour of objects and the structure of
+By communicating to users that the travel company is implementing a solution to their biggest issues, there is an inherent risk if we fail to deliver. If the solution fails to meet expectation, customers may feel rightly disappointed. If these issues have been ongoing, those who expected an end to their problems may be especially upset and cease interaction with the travel company. There is then  ...
 
-Not only this, but if the system does not work as intended, then the increase in operations (more tasks are performed as a result of the information system) may in fact work against the CMC and slow its efficiency, for little to no reward. This would produce a compounded negative effect, wherein not only is the CMC slower but it is expected to be faster. As such, the lengthened wait times and red ...
+Additionally, there is a small risk that the increase in tasks performed by CMC may slow its efficiency. This would provide a compounded effect, where the CMC is not only slower but expected to be faster too. This lower efficiency will lead to higher inbound call costs and reduced call flow handling.
```
Additionally, there is a small risk that the increase in tasks performed by CMC may decrease its effectiveness. This would have a run-on effect whereby despite the CMC being newly implemented it will be not only run slower but have less efficiency leading to higher inbound call costs and reduced call flow handling. (Just reworded it a bit)

seant1 replied:
BTW You can also suggest changes with that ± button. Then they can be committed/changed from the comment

zachartmann replied:
The CMC isn't newly implemented, right? I'll still take on the other feedback

zachartmann replied:
I'm not entirely sold either. My point in the second sentence was that they expected to improve its effectiveness but actually brought it down so the problem is twice as bad. Which, you know, can be cut I'm not attached to it or anything ahaha
Just double-checking that's the direction you think it should go

Aiswarya668 replied:
Yea definitely not newly implemented probably should've said improved. But what you have definitely works its all good

Aiswarya668 reviewed (changes requested):
Added a couple of changes - honestly just nitpicking at this stage. Thanks for the work you really made it more succinct! :)

zachartmann committed:
Minor changes due to feedback

* Fixed typos
* Deleted sentences
* Reworded statements

We're going to have to fix the names of the collaboration diagrams too so they show up properly

zachartmann committed:
Word change

Chunk to portion

Co-authored-by: seant1 <45996296+seant1@users.noreply.github.com>

zachartmann committed:
Formality typo

Because of -> Due to

Co-authored-by: seant1 <45996296+seant1@users.noreply.github.com>

zacaz90 commented:
Sorry I missed a few changes before asking for re-review. If you get here quickly just wait a second ahaha.

zachartmann committed:
Further shortening

Combined two sentences

Co-authored-by: seant1 <45996296+seant1@users.noreply.github.com>

zachartmann committed:
Slight wording enhancement

* Slowed efficiency -> Decreased effectiveness

zacaz90 commented:
Alright thanks for waiting Sean x

Aiswarya668 reviewed (approved):
Great job Zac - don't worry about anything else I said I totally agree with the points you raise :)
